cmd/pulumi-converter-terraform/main.go

```diff
@@ -107,15 +108,24 @@ func (*tfConverter) ConvertProgram(_ context.Context,
 				return translatedExample{}, fmt.Errorf("read example %s from memory store: %w", name, err)
 			}
 
+			yml, err := afero.ReadFile(dst, "/Pulumi.yaml")
+			if err != nil && !os.IsNotExist(err) {
+				return translatedExample{}, fmt.Errorf("read example Pulumi.yaml %s from memory store: %w", name, err)
+			}
+
 			return translatedExample{
 				PCL:         string(pcl),
+				YAML:        string(yml),
```

Comment:
Minor paranoia: would we ever look at generating things like `Pulumi.deployment.yaml`s etc. or stack-specific YAMLs? If so then maybe we'd call this `pulumiYAML`? But if not then ignore me; looks good.

Reply:
Probably not from the converter but happy to rename it just in case